best.ellie.StartupConfiguration.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "best.ellie.StartupConfiguration",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "startup-configuration",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/autostart:create",
+    "--filesystem=host:ro",
+    "--filesystem=home/.local/share:ro"
```

Comment:
These should be more specific, `~/.local/share/applications`, `~/.local/share/flatpak` and so on